atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemaTest.java

```diff
@@ -133,8 +134,26 @@ public void testLongTableNameLengthFailsNamespace() throws IOException {
                 .hasMessage(getErrorMessage(longTableName, kvsList));
     }
 
+    @Test
+    // If you are intentionally making Table API changes, please manually regenerate the ApiTestSchema
+    public void checkAgainstAccidentalTableAPIChanges() throws IOException {
+        // TODO (amarzoca): Add tests for schemas that use more of the rendering features (Triggers, StreamStores, etc)
+        Schema schema = ApiTestSchema.getSchema();
+        schema.renderTables(testFolder.getRoot());
+
+        String generatedTestTableName = "SchemaApiTestTable";
+        String generatedFilePath =
+                String.format("com/palantir/atlasdb/table/description/generated/%s.java", generatedTestTableName);
+
+        File expectedFile = new File("src/test/java", generatedFilePath);
```

Comment:
The expected file path here is very long and contains generated making it difficult to realise what the file is. Generally we want to have such files in the resources folder so generateFilePath can be something like
`"/resources/generatedCodeForTest" + generatedTestTableName`.